projects/go-lib/src/lib/components/go-copy/go-copy.component.scss

```diff
@@ -0,0 +1,4 @@
+.hideTextArea {
```

Comment:
If we update the markup like in the previous comment this would become: `.go-copy__textarea`